examples/configs/supported.py

```diff
@@ -29,10 +30,12 @@
     'poverty': PovertyMapDataset,
     'fmow': FMoWDataset,
     'bdd100k': BDD100KDataset,
+    'py150': Py150Dataset,
 }
 
 losses = {
     'cross_entropy': ElementwiseLoss(loss_fn=nn.CrossEntropyLoss(reduction='none')),
+    'lm_cross_entropy': ElementwiseLoss(loss_fn=lm_cross_entropy_loss),
```

Comment:
Since ElementwiseLoss is supposed to return loss tensor of size (batch size, ), I don't think it works well for this task. Can we maybe use MultiTaskLoss instead?